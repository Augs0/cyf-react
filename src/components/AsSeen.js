import React from 'react'

export const AsSeen = () => (
  <div className="container">
    <div className="row">
      <div className="col-sm-6 col-md-6">
          <h2>As seen in</h2>
          <div className="divider-1"><span></span></div>
          <br />
      </div>
    </div>
    <div className="row">
      <div className="col-sm-3">
          <a href="http://www.wired.co.uk/article/codeyourfuture-refugee-coding-school"
            target="_blank"
            rel="noopener noreferrer">
            <div className="icon--large">
              <img src="img/media-logos/wired.svg" alt="Wired"/>
            </div>
          </a>
      </div>
      <div className="col-sm-3">
          <a href="http://www.unhcr.org/news/stories/2017/1/586e420c7/volunteers-train-refugees-to-crack-into-london-tech-industry.html"
            target="_blank"
            rel="noopener noreferrer">
            <div className="icon--large">
              <img src="img/media-logos/unhcr.png" alt="UNHCR"/>
            </div>
          </a>
      </div>
      <div className="col-sm-3">
          <a href="https://www.newsdeeply.com/refugees/articles/2016/10/19/welcome-to-londons-refugee-coding-school"
            target="_blank"
            rel="noopener noreferrer">
            <div className="icon--large">
              <img src="img/media-logos/refugees-deeply.jpeg" alt="Refugees Deeply" />
            </div>
          </a>
      </div>
      <div className="col-sm-3">
          <a href="http://www.bbc.co.uk/programmes/p04yzrrg"
            target="_blank"
            rel="noopener noreferrer">
            <div className="icon--large">
              <img src="img/media-logos/BBC.svg" alt="BBC" />
            </div>
          </a>
      </div>
    </div>
  </div>
)