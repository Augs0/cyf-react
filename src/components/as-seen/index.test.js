import React from 'react';
import { shallow } from 'enzyme';
import AsSeen from './';

describe('AsSeen', () => {
  it('renders', () => {
    expect(shallow(<AsSeen />)).toMatchSnapshot();
  });
});
