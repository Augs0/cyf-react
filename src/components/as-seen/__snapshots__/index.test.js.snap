// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AsSeen renders 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col-sm-6 col-md-6"
    >
      <h2>
        As seen in
      </h2>
      <div
        className="divider-1"
      >
        <span />
      </div>
      <br />
    </div>
  </div>
  <div
    className="row"
  >
    <Outlet
      alt="Wired"
      key="img/media-logos/wired.svg"
      src="img/media-logos/wired.svg"
      url="http://www.wired.co.uk/article/codeyourfuture-refugee-coding-school"
    />
    <Outlet
      alt="UNHCR"
      key="img/media-logos/unhcr.png"
      src="img/media-logos/unhcr.png"
      url="http://www.unhcr.org/news/stories/2017/1/586e420c7/volunteers-train-refugees-to-crack-into-london-tech-industry.html"
    />
    <Outlet
      alt="Refugees Deeply"
      key="img/media-logos/refugees-deeply.jpeg"
      src="img/media-logos/refugees-deeply.jpeg"
      url="https://www.newsdeeply.com/refugees/articles/2016/10/19/welcome-to-londons-refugee-coding-school"
    />
    <Outlet
      alt="BBC"
      key="img/media-logos/BBC.svg"
      src="img/media-logos/BBC.svg"
      url="http://www.bbc.co.uk/programmes/p04yzrrg"
    />
  </div>
</div>
`;
